pom.xml

```diff
@@ -89,6 +89,10 @@ under the License.
   </pluginRepositories>
 
   <dependencies>
+    <dependency>
```

Comment:
Why add the dependency here and exclude from `apache-httpcomponents-client-4-api` that already have the dependency? Why not add the `commons-lang3-api` at `apache-httpcomponents-client-4-ap`?

Reply:
> Why […] exclude from `apache-httpcomponents-client-4-api`

I think you have misread the diff because the exclusion is from `pac4j-saml`, not from `apache-httpcomponents-client-4-api`.